playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -423,3 +423,14 @@
   tags:
     - manage
     - deploy:platform
+
+- name: Create OAuth credentials
```

Comment:
@mrgnr maybe we can use django_manage here. This is correct, but maybe is clear to use the ansible specific command.